tools/vuln_valid/index.js

```diff
@@ -24,7 +24,14 @@ const npmModel = joi.object().keys({
   updated_at: joi.string().regex(/^\d{4}-\d{2}-\d{2}$/).required().isoDate(),
   title: joi.string().required(),
   title: joi.string().max(150).regex(/^[^\n]+$/).required(),
-  author: joi.string().allow(null).required(),
+  author: joi.object().required(),
```

Comment:
the first author field is redundant, no?
same goes for the title field above

Reply:
You where right, I removed the first `author: joi.object().required()`. As for the `title` field. I don't know what the intention with this was. But I can clean this up in a separate PR. I think there might be more in this file that could need a touch up.

Reply:
Sure, np